yarn-project/aztec.js/src/utils/cheat_codes.ts

```diff
@@ -267,9 +267,9 @@ export class AztecCheatCodes {
     await this.eth.setNextBlockTimestamp(to);
     // also store this time on the rollup contract (slot 2 tracks `lastBlockTs`).
     // This is because when the sequencer executes public functions, it uses the timestamp stored in the rollup contract.
-    await this.eth.store(rollupContract, 7n, BigInt(to));
+    await this.eth.store(rollupContract, 6n, BigInt(to));
```

Comment:
in favor of constant names for these at the top of the file

Reply:
I'm nuking this function from orbit in the thing I'm working on atm for constraining based on slots.